frontend/src/components/VAudioThumbnail/meta/VAudioThumbnail.stories.js

```diff
@@ -0,0 +1,72 @@
+import VAudioThumbnail from "~/components/VAudioThumbnail/VAudioThumbnail.vue"
```

Comment:
It's a shame to add so much untyped and unchecked JavaScript back to the codebase after we spent so much effort converting everything to TypeScript. This was, after all, the main thing I argued against using MDX in the first place for :confused: 

Can you open either a single new issue or multiple issues for us to go through and convert these to TypeScript after the Nuxt 3 migration is finished? Storybook supports it, apparently without any fuss: https://storybook.js.org/docs/writing-stories/typescript

Reply:
Issue opened: #4290